Bug: /v3/console/ai/mcp/importToolsFromMcp fails in 3.2 due to json-schema-validator dependency conflict

## Bug description

In Nacos 3.2, calling `/v3/console/ai/mcp/importToolsFromMcp` can fail with a `NoClassDefFoundError`:

```text
Caused by: jakarta.servlet.ServletException: Handler dispatch failed: java.lang.NoClassDefFoundError: com/networknt/schema/SpecVersion$VersionFlag
    at org.springframework.web.servlet.DispatcherServlet.doDispatch(DispatcherServlet.java:1104)
    ... 117 common frames omitted
Caused by: java.lang.NoClassDefFoundError: com/networknt/schema/SpecVersion$VersionFlag
    at io.modelcontextprotocol.json.schema.jackson.DefaultJsonSchemaValidator.<init>(DefaultJsonSchemaValidator.java:47)
    at io.modelcontextprotocol.json.schema.jackson.DefaultJsonSchemaValidator.<init>(DefaultJsonSchemaValidator.java:42)
    at io.modelcontextprotocol.json.schema.jackson.JacksonJsonSchemaValidatorSupplier.get(JacksonJsonSchemaValidatorSupplier.java:26)
    at io.modelcontextprotocol.json.schema.jackson.JacksonJsonSchemaValidatorSupplier.get(JacksonJsonSchemaValidatorSupplier.java:17)
    at io.modelcontextprotocol.json.schema.JsonSchemaInternal.lambda$createDefaultValidator$1(JsonSchemaInternal.java:55)
    at java.base/java.util.stream.ReferencePipeline$7$1.accept(ReferencePipeline.java:273)
    at java.base/java.util.stream.Streams$StreamBuilderImpl.tryAdvance(Streams.java:397)
    at java.base/java.util.stream.ReferencePipeline$7$1.accept(ReferencePipeline.java:280)
    at java.base/java.util.ServiceLoader$ProviderSpliterator.tryAdvance(ServiceLoader.java:1499)
    at java.base/java.util.stream.ReferencePipeline.forEachWithCancel(ReferencePipeline.java:129)
    at java.base/java.util.stream.AbstractPipeline.copyIntoWithCancel(AbstractPipeline.java:527)
    at java.base/java.util.stream.AbstractPipeline.copyInto(AbstractPipeline.java:513)
    at java.base/java.util.stream.AbstractPipeline.wrapAndCopyInto(AbstractPipeline.java:499)
    at java.base/java.util.stream.FindOps$FindOp.evaluateSequential(FindOps.java:150)
    at java.base/java.util.stream.AbstractPipeline.evaluate(AbstractPipeline.java:234)
    at java.base/java.util.stream.ReferencePipeline.findFirst(ReferencePipeline.java:647)
    at io.modelcontextprotocol.json.schema.JsonSchemaInternal.createDefaultValidator(JsonSchemaInternal.java:61)
    at io.modelcontextprotocol.json.schema.JsonSchemaInternal.getDefaultValidator(JsonSchemaInternal.java:30)
    at io.modelcontextprotocol.json.schema.JsonSchemaValidator.getDefault(JsonSchemaValidator.java:61)
    at io.modelcontextprotocol.client.McpClient$SyncSpec.build(McpClient.java:478)
    at com.alibaba.nacos.console.controller.v3.ai.ConsoleMcpController.importToolsFromMcp(ConsoleMcpController.java:167)
```

## Root cause

This appears to be caused by a `json-schema-validator` dependency conflict.

Dependency chain:

```text
nacos-console
  ├── mcp:0.14.1
  │     └── mcp-json-jackson2:0.14.1
  │           └── json-schema-validator:1.5.7
  │
  └── nacos-copilot
        └── agentscope-core:1.0.7
              └── json-schema-validator:3.0.0
```

The MCP SDK expects `json-schema-validator:1.5.7`, which still contains `com.networknt.schema.SpecVersion$VersionFlag`.

However, `agentscope-core:1.0.7` brings in `json-schema-validator:3.0.0`. Starting from `2.0.0`, this library introduced a breaking API refactor and removed `SpecVersion$VersionFlag`.

After Maven dependency mediation selects `3.0.0`, MCP SDK code tries to load `SpecVersion$VersionFlag` and fails with:

```text
java.lang.ClassNotFoundException: com.networknt.schema.SpecVersion$VersionFlag
```

## Suggested fix

Please align the `json-schema-validator` version used by the MCP-related path and `agentscope-core`, for example by:

1. Forcing a compatible `json-schema-validator` version for the MCP SDK path.
2. Excluding the incompatible transitive dependency from `agentscope-core`.
3. Upgrading the MCP SDK to a version compatible with `json-schema-validator` 3.x, if available.

## Impact

This prevents `/v3/console/ai/mcp/importToolsFromMcp` from working in 3.2 under the conflicting dependency set.

--- a/pom.xml
+++ b/pom.xml
@@ -154,7 +154,7 @@
         <SnakeYaml.version>2.0</SnakeYaml.version>
         <kubernetes.client.version>22.0.0</kubernetes.client.version>
         <junit5.version>5.10.2</junit5.version>
-        <mcp.version>0.14.1</mcp.version>
+        <mcp.version>0.17.0</mcp.version>
         <ojdbc.version>19.7.0.0</ojdbc.version>
 
         <!-- Maven Central Portal -->

